src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_inpaint.py

```diff
@@ -5,6 +5,7 @@
 import torch
 
 import PIL
+from torchvision.transforms.functional import to_pil_image
```

Comment:
Sorry we cannot use this here -> we don't have a dependency on `torchvision` . Could you instead use the `pipe.numpy_to_pil` function?

Reply:
Ah right, I thought it would be OK as it is listed as a dependency [here](https://github.com/huggingface/diffusers/blob/a4802294630b823125a836c8d02b83aac13eab6a/setup.py#L105) (though maybe I have misunderstood this file. I will change this.

Reply:
Hey @vvvm23,

That's a good point. The list you linked as a list of "soft-dependencies" which are not required for core functionality but for certain use cases which in the case of `torchvision` is training. 

The "hard-dependencies" can be found here: https://github.com/huggingface/diffusers/blob/a4802294630b823125a836c8d02b83aac13eab6a/setup.py#L201

Reply:
Maybe we could instead make `prepare_mask_and_masked_image` a method of the pipeline and instead use `self.numpy_to_pil`?

Reply:
I think it being in or out the pipeline would work, as numpy to pil is a static method so can be accessed without actually passing a reference to the pipe. 

I prefer it as a method of the pipe I'll try and get something to you tomorrow ~

Reply:
We still cannot have a dependency on `torchvision` though ;-)

Reply:
I removed the `torchvision` dependency.

I noticed also the docstrings make reference to accepting batched inputs. So I'll add that too soon

Reply:
Actually, not sure if adding batching is easily doable here. Might be worth having a different issue for that. Can you take a look and see what you think?

I'll fix the conflicts for what I have so far.